### PR TITLE
feat(profiling) construct profile from continuous chunk

### PR DESCRIPTION
**Pull request by JonasBa**
This PR implements measurements rendering for the continuous profiling UI.

The chart to render slow and frozen frames still needs to be updated to reflect this.

### CODE REVIEW

File: static/app/utils/profiling/flamegraphChart.tsx [hunk 1]
```diff
@@ -115,14 +118,14 @@ export class FlamegraphChart {
         }
 
         // Track and update X domain max and min
-        if (m.elapsed_since_start_ns > this.domains.x[1]) {
-          this.domains.x[1] = m.elapsed_since_start_ns;
+        if (m.elapsed > this.domains.x[1]) {
+          this.domains.x[1] = m.elapsed;
```
Comment (JonasBa) [inline static/app/utils/profiling/flamegraphChart.tsx:122]: this had previously shared the type with the measurement, but was now moved to a separate field

File: static/app/components/profiling/flamegraph/continuousFlamegraph.tsx [hunk 2]
```diff
@@ -229,7 +232,11 @@ export function ContinuousFlamegraph(): ReactElement {
     }
     return new UIFrames(
       {
+        // @TODO
+        // @ts-expect-error
```
Comment (JonasBa) [inline static/app/components/profiling/flamegraph/continuousFlamegraph.tsx:236]: these need to be updated, I'm going to tackle them in the next PR

File: static/app/components/profiling/flamegraph/continuousFlamegraph.tsx [hunk 3]
```diff
@@ -251,25 +258,31 @@ export function ContinuousFlamegraph(): ReactElement {
...
-            };
-          }),
-          // some versions of cocoa send byte so we need to correct it to watt
-          unit: 'watt',
-          name: 'CPU energy usage',
-        });
+        const measurements = profileGroup.measurements[key]!;
+        const values: ProfileSeriesMeasurement['values'] = [];
+
+        let offset = 0;
```
Comment (JonasBa) [inline static/app/components/profiling/flamegraph/continuousFlamegraph.tsx:264]: This computes the value offset to the start of the profile

Comment (Zylphrex) [reply static/app/components/profiling/flamegraph/continuousFlamegraph.tsx:264]: Might be good to put into a helper function that takes a formatted for the value? I see it's copied for all the other charts below.

**Review by Zylphrex — APPROVED** [review]

File: static/app/components/profiling/flamegraph/continuousFlamegraph.tsx [hunk 4]
```diff
@@ -123,7 +126,7 @@ export function ContinuousFlamegraph(): ReactElement {
   const dispatch = useDispatchFlamegraphState();
 
   const profiles = useContinuousProfile();
-  const profileGroup = useProfileGroup();
+  const profileGroup = useProfileGroup() as ContinuousProfileGroup;
```
[the remark this answers is not in the document]
Comment (JonasBa) [reply static/app/components/profiling/flamegraph/continuousFlamegraph.tsx:129]: Let me tackle this in a separate PR with the approach we discussed (separate context and asserting data shape). We can get rid of calling useProfiles there too

**Pull request merged**